jekyll/_cci2/monitoring.adoc

```diff
@@ -12,15 +12,64 @@ This section includes information on metrics for monitoring your CircleCI Server
+* Retroactively understand infrastructure-wide issues
+
+=== How Metrics Work in CircleCI Server
+
+Telegraf is the main component used for metrics collection in CircleCI Server. Telegraf is server software that brokers metrics data emitted by CircleCI services to data monitoring platforms such as Datadog or AWS CloudWatch.
+
+.Metrics
+image::metrics.png[Metrics]
+
+Metrics collection in CircleCI Server works as follows:
+
+* Each component of a Server installation sends metrics data to the telegraf container running on the Services machine.
+* Telegraf listens on port 8125/UDP and receives data from all components (inputs) and applies configured filters to determine whether data should be kept or dropped.
+* For some metric-types, Telegraf keeps metrics data inside and calculates statistical data (such as max, min, mean, stdev, sum) periodically.
+* Finally, Telegraf sends out data to stdout and configured data platforms (outputs), such as stdout, Datadog and/or AWS CloudWatch.
+
+It is worth noting that Telegraf can accept multiple input and output types at the same time allowing administrators to configure a single Telegraf instance to collect and forward multiple metrics data sets to both Datadog and CloudWatch.
+
+== Standard Metrics Configuration
+
+Review your metrics configuration file using the following command:
+
+{% raw %}
+```sh
+sudo docker inspect --format='{{range .Mounts}}{{println .Source "->" .Destination}}{{end}}' telegraf | grep telegraf.conf | awk '{ print $1 }' | xargs cat
+```
+{% endraw %}
+
+There are three notable blocks in the file (some blocks might not be there depending on your configuration in the Management Console):
+
+* `\[[inputs.statsd]]` – Input configuration to receive metrics data through 8125/UDP (as discussed above)
+* `\[[outputs.file]]` – Output configuration to emit metrics to stdout. All accepted metrics are configured to be shown in Telegraf docker logs. This is helpful for debugging your metrics configuration.
+* `\[[outputs.cloudwatch]]` – Output configuration to emit metrics to CloudWatch
+* `\[[outputs.datadog]]` – Output configuration to emit metrics to Datadog
+
+This configuration file is automatically generated by Replicated (the service used to manage and deploy CircleCI Server) and is fully managed by Replicated. If you wish to customize the standard configuration you will need to configure Replicated to **not** insert the blocks you want to change. 
+
+CAUTION: Do not attempt to directly modify the file. Any changes made in this way will be destroyed by Replicated upon certain events, such as service restarts. For example, if a customized `\[[inputs.statsd]]` block is added _without_ stopping automatic interpolation, you will encounter errors as Telegraf attempts to listen to `8125/UDP` twice, and the second attempt will fail with `EADDRINUSE`.
+
+In a standard configuration (with no metrics customization) the main config discussed above is for Telegraf. 
```

Comment:
With this sentence I implied that there is nothing else but the main config to care about, if there is no metrics customization. I just want to make sure that the implication is conveyed with this wording. Thanks!